### DISCUSSION TITLE
Ever tried to deploy on azure static website

### DISCUSSION THREAD

**Discussion by sebastienminne** [quasarframework/quasar#16148] [General - Components / Directives / etc]
Hello there, 

Any chance one of you tried to deploy a quasar app onto azure static website ? 
Can't find any topics about this. A shame azure offer a free way to expose static webapp but ... seems impossible to host a quasar apps

**Comment by stefanvanherwijnen** [comment]
You will need to use a static site generator, e.g. https://github.com/freddy38510/quasar-app-extension-ssg.
